src/wallet/asyncrpcoperation_common.cpp

```diff
@@ -179,6 +179,15 @@ void ThrowInputSelectionError(
                     WEIGHT_RATIO_CAP,
                     FormatMoney(err.conventionalFee)));
         },
+        [](const MaxFeeError& err) {
+            throw JSONRPCError(
+                RPC_INVALID_PARAMETER,
+                strprintf(
+                    "Fee %s is greater than the maximum fee allowed by this instance (%s). Run "
+                    "zcashd with `-maxtxfee` to adjust this limit.",
+                    FormatMoney(err.fixedFee),
+                    FormatMoney(maxTxFee)));
+        },
```

Comment:
The default `-maxtxfee` is 0.1 ZEC (`DEFAULT_TRANSACTION_MAXFEE` in `main.h`). This seems reasonable because it corresponds to a ZIP 317 fee with 10000000/5000 = 2000 logical actions.

Reply:
I agree that it’s reasonable, but with maximum prioritisation, it’s “only” 500 logical actions.

Reply:
Keep in mind that prioritization only really starts to become relevant when blocks are full of transactions paying the conventional fee, and people then decide to give up additional privacy by sending larger fees to get an advantage. Champagne problem, let's revisit that constant after 5.7.0 or something. :)